src/styles/_catalog.scss

```diff
@@ -1,22 +1,30 @@
+$card-width: 200px;
```

Comment:
You can create a new file for variables even if you have only one like in this case I think.

Reply:
I have file for variables, or you mean smth else?